--- config.py
+++ config_new.py
@@ -30,6 +30,7 @@
 import socket
 import subprocess
 from libqtile.config import Drag, Key, Screen, Group, Drag, Click, Rule
+from libqtile.config import ScratchPad, DropDown
 from libqtile.command import lazy, Client
 from libqtile import layout, bar, widget, hook
 from libqtile.widget import Spacer
@@ -59,7 +60,7 @@
 
 # FUNCTION KEYS
 
-    Key([], "F12", lazy.spawn('xfce4-terminal --drop-down')),
+    #Key([], "F12", lazy.spawn('xfce4-terminal --drop-down')),
 
 # SUPER + FUNCTION KEYS
 
@@ -296,6 +297,20 @@
         Key([mod, "shift"], i.name, lazy.window.togroup(i.name) , lazy.group[i.name].toscreen()),
     ])
 
+groups.append(
+    ScratchPad("scratchpad", [
+        # define a drop down terminal.
+        # it is placed in the upper third of screen by default.
+        #DropDown("atlassian", "firefox", opacity=0.88, height=0.55, width=0.80, ),
+        DropDown("term", "/usr/bin/urxvt", opacity=0.88, height=0.55, width=0.80, ),
+]), )
+
+keys.extend([
+    # Scratchpad
+    # toggle visibiliy of above defined DropDown named "term"
+    #Key([], 'F11', lazy.group['scratchpad'].dropdown_toggle('atlassian')),
+    Key([], 'F12', lazy.group['scratchpad'].dropdown_toggle('term')),
+])
 
 def init_layout_theme():
     return {"margin":5,

